.github/workflows/dev.yaml

```diff
@@ -68,6 +69,7 @@ jobs:
           - hbase
           - hive
           - kafka
+          - krb5
```

Comment:
This image doesn't use ubi8 so it won't pass the preflight checks. This can be removed,

Reply:
You're absolutely right, thx!